cis194/week1/wei-ning/src/Assert.hs

```diff
@@ -0,0 +1,8 @@
+module Assert (assert) where
+
+assert :: String -> Bool -> IO ()
+assert msg expr =
+  if expr
+  then putStrLn (msg ++ " : passed")
+  else error    (msg ++ " : failed")
```

Comment:
@shineli1984 in Haskell, is if then else encouraged?

Reply:
how about?
```
assert msg True  = putStrLn (msg ++ " : passed")
assert msg False = error    (msg ++ " : failed")
```

Reply:
Yeah can do.
"Prefer guards over if-then-else where possible." - haskell style guide https://kowainik.github.io/posts/2019-02-06-style-guide#if-then-else-clauses